src/app/data-entries/dropdown/dropdown-menu/dropdown-menu.component.scss

```diff
@@ -1,31 +1,35 @@
-.drop-down-wrapper{
-    display: flex;
-    flex-direction: column;
-    padding-top: 5px;
-}
+::ng-deep {
```

Comment:
No need to `ng-deep` all the rules, just the ones that affect child elements not in the component view.